docs/v3/concepts/artifacts.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Create run artifacts
```

Comment:
same question as other PR about concept titles

Reply:
Updated to be simply "Artifacts"